ssm/helpers.go

```diff
@@ -39,16 +39,23 @@ func CreateSSMDescribeInstanceInput(filters map[string]string, instances CommaSl
 	return ssmInput
 }
 
-func addInstanceInfo(instanceID *string, tags map[string]string, instancePool *instance.InstanceInfoSafe, profile string, region string) {
+func addInstanceInfo(instanceID *string, ec2Instance ec2.Instance, instancePool *instance.InstanceInfoSafe, profile string, region string) {
```

Comment:
Strips out the individual tags and attributes in favor of formatting to the struct from within this method.